### PR TITLE
Update seed function for 10M entries

### PR DESCRIPTION
**Pull request by kwang429**

### CODE REVIEW

File: database/pgSeed.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  'Mirror',
+  'Stool',
+  'Bench',
+  'Curtain',
+  'Organizer',
+  'Bowl',
+  'Cutlery'
+];
+const dimensions = [
+  'null',
```
Comment (Stevejrmc) [inline database/pgSeed.js]: I'm curious, why is there a null dimension? Not a critique, genuinely don't know.

File: database/pgSeed.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  function writeFile() {
+    let ok = true;
+    do {
+      i--;
+      // write the very last entry
+      if (i === 0) {
+        stream.write(createProduct(10000000 - i));
+        stream.end();
+      } else {
+        ok = stream.write(createProduct(10000000 - i) + '\n');
```
Comment (Stevejrmc) [inline database/pgSeed.js]: More curiosity: What is the intention when reseting 'ok' here?

File: database/pgSeed.js (hunk 1 above)
Comment (kwang429) [reply database/pgSeed.js]: i saw that some products on the ikea website don't show their dimensions in their subheading (at least in the search bar) - so i included a null so some of them won't display that detail

File: database/pgSeed.js (hunk 2 above)
Comment (kwang429) [reply database/pgSeed.js]: i got this format straight from the stream.write example on node and if i understand it correctly, it's just constantly checking if stream.write succeeded.
so if it fails and ok evaluates to false, it'll break out of the while loop and go into the if statement on line 98

Comment (Stevejrmc) [reply database/pgSeed.js]: Thanks for sharing.

**Pull request merged**